x-pack/plugins/enterprise_search/server/index.ts

```diff
@@ -17,7 +17,10 @@ export const plugin = (initializerContext: PluginInitializerContext) => {
 export const configSchema = schema.object({
   accessCheckTimeout: schema.number({ defaultValue: 5000 }),
   accessCheckTimeoutWarning: schema.number({ defaultValue: 300 }),
+  canDeployEntSearch: schema.boolean({ defaultValue: true }),
```

Comment:
Not sure whether these should be true by default. Not sure they shouldn't be either. 🤔

Reply:
My thought there was I want to keep the default behavior matching current app. Then we will only set these to `false` in the config when running `serverless_search`